mcf-browser/src/back-end/parse-tmcf.js

```diff
@@ -23,9 +23,10 @@ import {shouldReadLine} from './utils.js';
  * @param {string} propValue The string to look for a column name in.
  * @return {string|null} The column name that comes after '->'.
```

Comment:
This line and "Ex: C:SomeDataset->GeoId would return 'GeoId'" above needs update?